client/app/pages/queries/query.html

```diff
@@ -102,7 +102,8 @@
               <span class="text-muted">Data Source</span>
               <select ng-disabled="!isQueryOwner" ng-model="query.data_source_id" ng-change="updateDataSource()"
                       ng-options="ds.id as ds.name for ds in dataSources"></select>
-              <a ng-if="dataSource.options.doc_url" ng-href={{dataSource.options.doc_url}}>{{dataSource.type_name}} documentation</a>
+              <a ng-if="dataSource.options.doc_url != ''" ng-href={{dataSource.options.doc_url}}>{{dataSource.type_name}} documentation</a>
+              <span ng-if="dataSource.options.doc_url ==''">{{dataSource.type_name}}</span>
```

Comment:
This line and the one below it seem logically equivalent. Should one be removed? Also, by convention in this file a space should be added after the `==`.